### PR TITLE
Effortsheet cubejs

### PR DESCRIPTION
**Pull request by AkhileshNegi**
Cube_JS integration to show project monthly billable hours

### CODE REVIEW

File: resources/views/knowledgecafe/library/books/show.blade.php [hunk 1]
```diff
@@ -95,6 +95,9 @@
         :user = "{{ auth()->user() }}"
         />
     </div>
+    <div class="mt-3 w-75">
+        <effort-component/>
```
Comment (P4NK4J) [inline resources/views/knowledgecafe/library/books/show.blade.php:99]: @AkhileshNegi Is there any specific use case of using the KnowledgeCafe module in effortsheet integration with Cube.Js? Let me know if I am missing something.

**Review by P4NK4J — COMMENTED** [review]
Left a comment, please take a look.

File: resources/views/knowledgecafe/library/books/show.blade.php (hunk 1 above)
Comment (AkhileshNegi) [reply resources/views/knowledgecafe/library/books/show.blade.php]: @P4NK4J No there isn't. It's a miss from my end, I have reverted changes to it. Thanks

**Review by GauravGusain98 — APPROVED** [review]
Looks fine

**Pull request merged**